it_IT/ODF-community.md

```diff
@@ -23,18 +23,18 @@ Contenuti
 ===========================================================================
 * [Panoramica](#panoramica)
   * [Che cos'è una decisione aperta?](#what-is-an-open-decision)
-  * [Le decisioni aperte sono prese secondo i principi Open Source](#open-decisions-are-made-using-open-source-principles)
```

Comment:
I do not know at this point the real meaning of the English assertion. I thought about the decision-making process. That is, "Open decisions are taken through an open approach" and not that their content speaks about this approach ...
But mind, I'm a base english speaker :)

Reply:
Right, even better would be "Il processo decisionale aspetto rispecchia i principi open source". I liked the use of the active voice, passive sounds really bad in this sentence.

 I will shortly push an update.